global.json

```diff
@@ -1,4 +1,7 @@
 {
+  "sdk": {
+    "version": "2.2.199"
```

Comment:
If this is not here, then the build will fail. Important that it be not the same as the actual .NET Core installed as that also fails. See more here: https://github.com/dotnet/sdk/issues/3376#issuecomment-529007198